packages/components/src/radio-group/stories/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Radio from '../../radio';
+import RadioGroup from '../';
+
+export default { title: 'Components/RadioGroup', component: RadioGroup };
+
+export const _default = () => {
+	/* eslint-disable no-restricted-syntax */
+	// id is required for server side rendering
+	return (
+		<RadioGroup
+			id="default-radiogroup"
+			accessibilityLabel="options"
+			defaultChecked="option2"
+		>
+			<Radio id="option1-radio" value="option1">
+				Option 1
+			</Radio>
+			<Radio id="option2-radio" value="option2">
+				Option 2
+			</Radio>
+			<Radio id="option3-radio" value="option3">
+				Option 3
+			</Radio>
```

Comment:
If this is for server side rendering, you don't need to explicitly pass the ids to the individual radio buttons.
```suggestion
			<Radio value="option1">Option 1</Radio>
			<Radio value="option2">Option 2</Radio>
			<Radio value="option3">Option 3</Radio>
```

Reply:
Good catch. I had assumed, incorrectly, that since it was required for `RadioGroup` it would be the same for `Radio`. Fixed in da54def.